src/content/docs/en/guides/backend/turso.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Turso & Astro
+description: Build locally with a SQLite file and deploy globally using Turso.
+type: backend
+service: Turso
+stub: false
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[Turso](https://turso.tech) is a distributed database built on libSQL, a fork of SQLite. It is optimized for low query latency, making it suitable for global applications.
+
+### Prerequisites
+
+- The [Turso CLI](https://docs.turso.tech/reference/turso-cli) installed and signed in
+- A [Turso](https://turso.tech) Database with schema
+- Your Database URL
+- An Access Token
+
+If you don't have an existing database and schema with Turso, create one and connect to the shell using the Turso CLI:
+
+```bash
+turso db create my-astro-db
+turso db shell my-astro-db
+```
+
+Then add the following:
+
+```sql
+CREATE TABLE posts (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    title TEXT,
+    slug TEXT UNIQUE,
+    content TEXT
+);
+```
+
+Populate the `posts` table with some initial content by **[DOING THIS THING, IDEALLY LINKING TO TURSO DOCS]**.  This is required so you can write queries and have data to fetch.
```

Comment:
```suggestion
```
Sounds like the intention was to remove this?

Reply:
Ha! Good spot. Totally missed this earlier rushing to get a reply back to you.